works/models.py

```diff
@@ -237,9 +238,9 @@ def __str__(self):
 
     def save(self, *args, **kwargs):
         if self.pk is None:
-            self.start_date = datetime.datetime.now()
+            self.start_date = timezone.now()
```

Comment:
I read that is better to use timezone than datetime because it gives you the date and time according to the server configuration.